crates/configuration/src/lib.rs

```diff
@@ -44,7 +44,7 @@ pub fn get_profile(
 ) -> Result<serde_json::Value> {
     let profile_name = profile.as_deref().unwrap_or("default");
     let tool_config = get_with_ownership(raw_config, tool)
-        .context("Failed to find {tool} config in snfoundry.toml file")?;
+        .unwrap_or(serde_json::Value::Object(serde_json::Map::new()));
```

Comment:
Is this a good idea? This may make the troubleshooting a bit harder, as we silently return an empty object instead of an error when someone specifies a profile that does not exist.

Reply:
I did this for consistency with the case of missing config. But I generally agree, I will change both.